js/src/util/index.ts

```diff
@@ -11,113 +11,118 @@ import {
     VersionedTransaction,
 } from '@solana/web3.js';
 
-export class NautilusUtils {

```

Comment:
Makes sense to not have these functions in a class for multiple reasons. Good stuff.

I wonder if we should try to make these private, ie. only exported to the `Nautilus` instance but not to anyone downstream?

Reply:
I feel like utility functions should be public because I used Anchor and Metaplex utils a lot